src/odemis/acq/stitching/_registrar.py

```diff
@@ -618,8 +620,8 @@ def _get_shift(self, prev_tile, tile):
         prev_tile_roi = numpy.array(prev_tile)[t1:b1, l1:r1]
         tile_roi = numpy.array(tile)[t2:b2, l2:r2]
 
-        # Calculate the shift
-        shift = MeasureShift(tile_roi, prev_tile_roi)
+        # Calculate the shift ([:,:] is a hint how to specify column/row for debugging purposes)
+        shift = MeasureShift(tile_roi[:, :], prev_tile_roi[:, :])
```

Comment:
I am not sure if we should do that. Maybe it would be better to specify the shape of `tile_roi` and `prev_tile_roi` at the place where you create the array? @pieleric What do you think?

Reply:
This is just there fore debugging purposes. I had a black line in my samples and therefore needed to exclude the last column of every part on which the shift was measured. I like to leave it here so that for a future user looking and going through the code it is easy to find where one should implement this easily (for debuging).

Reply:
I would say how to access a column and row in an array does not need to be explicitly specified in the code. This I would place under common knowledge. What you are describing that there are black lines that need to be handled, is something you don't mention in you comment - so it does not provide additional information as it is now.

Reply:
I also don't think it's worthy to modify the code itself. It's probably not going to slow it down much, but that's more confusing than helpful.

Honestly, even if you've spent quite some time looking for this trick, the use case is so unlikely to be used again that I'm not convinced it's really worthy to to mention here. Actually, if the tiles need to be crop, you should just crop them before sending passing them to the registrar. If you still think it could be useful, I'd suggest to just add a separate comment reading like:
`# If you need to crop the tiles, you can do it here with the pattern tile_roi[t:-b, l:-r]`